Unfortunately, some test failures are relevant:

```
FAILED test/integration/objectstore/test_rucio_objectstore.py::test_tools[composite_output] - galaxy.tool_util.verify.interactor.JobOutputsError: Job in error state.. tool_id: composite_output, exit_code: None, stderr: .
Job in error state.. tool_id: composite_output, exit_code: None, stderr: .

FAILED test/integration/objectstore/test_rucio_objectstore.py::test_tools[composite_output_tests] - galaxy.tool_util.verify.interactor.JobOutputsError: Job in error state.. tool_id: composite_output_tests, exit_code: None, stderr: .
Job in error state.. tool_id: composite_output_tests, exit_code: None, stderr: .

FAILED test/integration/objectstore/test_rucio_objectstore.py::test_tools[metadata] - galaxy.tool_util.verify.interactor.JobOutputsError: Timed out after 60.25 seconds waiting on tool test run.
```

--- a/lib/galaxy/objectstore/examples/rucio_distributed.yml
+++ b/lib/galaxy/objectstore/examples/rucio_distributed.yml
@@ -20,8 +20,8 @@ backends:
     download_schemes:
       - rse: TEST
         scheme: file
-        ignore_checksum: False
+        ignore_checksum: false
     cache:
       path: database/object_store_cache_rucio
       size: 1000
-      cache_updated_data: True
+      cache_updated_data: true

--- a/test/integration/objectstore/_base.py
+++ b/test/integration/objectstore/_base.py
@@ -39,6 +39,9 @@ RUCIO_OBJECT_STORE_CONFIG = string.Template(
     upload_scheme: file
     register_only: false
     download_schemes:
+      - rse: ${rucio_rse}
+        scheme: file
+        ignore_checksum: false
     scope: galaxy
     host: http://${host}:${port}
     account: ${rucio_account}

